### PR TITLE
Fix Windows CI

### PR DESCRIPTION
**Pull request by dyollb**
Partially fixes issue #2396 

fix windows ci
- use torch==2.4.1 (see https://github.com/MIC-DKFZ/nnUNet/issues/2396#issuecomment-3162932241_
- activate windows action

normalize gaussian using numpy
- as noted [here](https://github.com/MIC-DKFZ/nnUNet/issues/2396#issuecomment-3163312603), normalization using torch produces 'Inf' for the test case with torch==2.4.0.
- doing the operation on the numpy array instead should not have any negative side-effects

specify folder 'nnunetv2' in pyproject.toml
- doesn't hurt. I noticed when I had temporary folders (e.g., 'venv/'), setuptools was not able to install the package properly
- add `.venv/` to the gitignore file. This is a common name for a virtual environment

note: this PR does not 'fix' the issue with 2.4.0. I tried accumulating the `predicted_logits` in predict_from_raw_data.py using float32, which also helped. But in the end I still got a dice of 0.008

### CODE REVIEW

**Comment by FabianIsensee** [comment]
Thanks! I merged this. Let's :crossed_fingers: that everything works now :-)
